lib/module.js

```diff
@@ -441,6 +441,8 @@ function stripBOM(content) {
   return content;
 }
 
+Module._stripBOM = stripBOM;
```

Comment:
I suggest moving stripBOM to lib/internal/module.js (to be created) and then require() it in lib/module.js and src/node.js.